docs/vi/index.md

```diff
@@ -0,0 +1,300 @@
+---
+Bố cục: Mặc định
+Tiêu đề: Học sâu
+Tác giả: Alfredo Canziani
+Tham khảo: Trang chủ
```

Comment:
You cannot translate the tags. These are read by my program. These `MD` files are the "source code" for my script, which converts them into `HTML` to be served as web pages.
Please, use 

```
---
layout: default
title: DEEP LEARNING
author: Alfredo Canziani
lang-ref: home
---
```

and replace "DEEP LEARNING" with Vietnamese. You need to add `lang: vi`, `translator:`, and `translation-date:` as well. Please, check again other languages, okay?
This is Korean, for example.

```
---
layout: default
title: 딥러닝
author: Alfredo Canziani
lang-ref: home
lang: ko
translation-date: 22 Mar 2020
translator: Gio
---
```

Reply:
Ok. thank you for your annoucement so much! I will correct it again.